domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/InterestDate.java

```diff
@@ -60,6 +60,21 @@ public InterestEndDateType getEndDateType() {
         return endDateType;
     }
 
+    @JsonIgnore
```

Comment:
hmm?

Reply:
We don't want it to be mapped into JSON.